Scripts/WikiUpdater.cs

```diff
@@ -326,15 +547,29 @@ private string ConvertTextToBbcode(string paragraph)
             var translatedPage = page.TranslatedPage;
 
             // Translate page names
-            translationPairs.Add(untranslatedPage.Name, translatedPage.Name);
+            translationPairs.TryAdd(untranslatedPage.Name, translatedPage.Name);
```

Comment:
Why has this been changed to `TryAdd` now? Surely it's an error for the script to want to add two pages with the same name.